src/course-outline/configure-modal/ConfigureModal.jsx

```diff
@@ -16,33 +16,123 @@ import { getCurrentItem } from '../data/selectors';
 import messages from './messages';
 import BasicTab from './BasicTab';
 import VisibilityTab from './VisibilityTab';
+import AdvancedTab from './AdvancedTab';
 
 const ConfigureModal = ({
   isOpen,
   onClose,
   onConfigureSubmit,
 }) => {
   const intl = useIntl();
-  const { displayName, start: sectionStartDate, visibilityState } = useSelector(getCurrentItem);
+  const {
+    displayName,
+    start: sectionStartDate,
+    visibilityState,
+    due,
+    isTimeLimited,
+    defaultTimeLimitMinutes,
+    hideAfterDue,
+    showCorrectness,
+    courseGraders,
+    category,
+    format,
+  } = useSelector(getCurrentItem);
   const [releaseDate, setReleaseDate] = useState(sectionStartDate);
   const [isVisibleToStaffOnly, setIsVisibleToStaffOnly] = useState(visibilityState === VisibilityTypes.STAFF_ONLY);
   const [saveButtonDisabled, setSaveButtonDisabled] = useState(true);
+  const [graderType, setGraderType] = useState(format == null ? 'Not Graded' : format);
+  const [dueDateState, setDueDateState] = useState('');
+  const [isTimeLimitedState, setIsTimeLimitedState] = useState(false);
+  const [defaultTimeLimitMin, setDefaultTimeLimitMin] = useState(30);
+  const [hideAfterDueState, setHideAfterDueState] = useState(false);
+  const [showCorrectnessState, setShowCorrectnessState] = useState(false);
+  const [isSubsection, setIsSubsection] = useState(category === 'sequential');
```

Comment:
```suggestion
  const [isSubsection, setIsSubsection] = useState(category === COURSE_BLOCK_NAMES.sequential.id);
```
You'll need to import `COURSE_BLOCK_NAMES` from `src/course-outline/constants.js`.

**Edit:** Why do we need state here, can't we just use a variable?
```suggestion
  const isSubsection = category === COURSE_BLOCK_NAMES.sequential.id
```

Reply:
This has not been addressed. You can remove `useState` completely.